### DISCUSSION TITLE
How do I setup port forwarding in Shared Network mode?

### DISCUSSION THREAD

**Discussion by SiwonLim** [utmapp/UTM#6079] [UTM Help (macOS)]
Configuration
* UTM Version: 4.2.5
* OS Version: 14.1.1
* Intel or Apple Silicon? Apple M1

I can only see port forwarding settings in 'Emulated VLAN' mode. How do I setup port forwarding in 'Shared Network' mode?

**Comment by joshmeads** [comment]
You don’t, it’s a shared network. You can access the hosts ports directly. 

You can’t use localhost though, you’ll have to use the hosts local ip address. Check ipconfig on windows or ifconfig on Linux.
